src/kixi/gwyn/gwyn.clj

```diff
@@ -68,13 +71,48 @@
                                                coll-pumps-attending)}))))
         ds/dataset)})
 
+(defn adjust-avg-num-pumps
+  "Takes in the properties dataset. Adjust the average number of pumps
+   attending the fire depending on the value of the standard deviation."
+  [properties-data]
+  (-> properties-data
+      (wds/add-derived-column :adjusted-avg-pumps [:sd-pumps-attending :avg-pumps-attending]
+                              (fn [sd avg] (if (> sd 2) ;; have to chose what this value should be
```

Comment:
I guess maybe you could look at the output to help decide this value. Once chosen (or 2 is used) it will need a little explanation in the docstring above, to avoid the use of "magic numbers"